CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ project(Descent3
 )
 
 option(FORCE_COLORED_OUTPUT "Always produce ANSI-colored compiler warnings/errors (GCC/Clang only; esp. useful with ninja)." OFF)
+option(FORCE_PORTABLE_INSTALL "Install all files into local directory defined by CMAKE_INSTALL_PREFIX" OFF)
```

Comment:
This install is only meant to be portable, should we set CMAKE_INSTALL_PREFIX to a path relative to the build directory by default when not set by the user ([like there](https://stackoverflow.com/a/16076855))? The default CMAKE_INSTALL_PREFIX is a system path and don't want game files to end up there